sdk/python/distillation/benchmarking/pipelines/conversation/hellaswag.yaml

```diff
@@ -0,0 +1,286 @@
+$schema: https://azuremlschemas.azureedge.net/latest/pipelineJob.schema.json
```

Comment:
Do we not have a prod pipeline already which can be used for evaluation submission? 

cc: @iamrk04